imports/ui/admin/Admin.html

```diff
@@ -41,6 +41,27 @@
                 <input required id="japaneseName" name="japaneseName" class="form-control" placeholder="Japanese League Name...">
             </div>
         </div>
-        <button type="submit" style="width: 25%" class="btn btn-primary">Add</button>
+        <button type="submit" class="btn btn-primary">Add Player</button>
```

Comment:
Can you add a "Are you sure?" dialogue?  I've had people accidentally press add before they were done.

Reply:
Sure